psi4/src/psi4/occ/manager.cc

```diff
@@ -165,9 +162,6 @@ void OCCWave::omp2_manager() {
         Process::environment.globals["OMP2 TOTAL ENERGY"] = Emp2L;
         Process::environment.globals["SCS-OMP2 TOTAL ENERGY"] = Escsmp2;
         Process::environment.globals["SOS-OMP2 TOTAL ENERGY"] = Esosmp2;
```

Comment:
I thought you wanted SCS/SOS not copied to `variables_`, because that should be qcbd's responsibility? I'm fine bringing SCS/SOS to variables_ if you are.

Reply:
At the point that it’s qcdb’s responsibility, the only things that psi needs to set are same-spin, opposite-spin, and singles energies. Going that far would necessitate rewriting the occ printing, which seemed a step too far for this round. So yes, please go for variables_ only

Reply:
At the point that it’s qcdb’s responsibility, the only things that psi needs to set are same-spin, opposite-spin, and singles energies. Going that far would necessitate rewriting the occ printing, which seemed a step too far for this round. So yes, please go for variables_ only

Reply:
So is there anything for me to do on this point? `OMP2 TOTAL ENERGY` is set in `variables_` on the line above, and you don't want me to set `SCS-OMP2 TOTAL ENERGY` or `SOS-OMP2 TOTAL ENERGY`.

Reply:
I think kill off all the remaining `P::e.globals`. Then the copy in proc.py will populate them from `variables_` and keep it in sync.